src/views/admin/TestListView.vue

```diff
@@ -450,7 +450,7 @@ export default {
       this.subIndex = 0; //reset subIndex when main idex change
     },
   },
-  created() {
+  async created() {
     this.$store
```

Comment:
You forgot **await** before this dispatch. You created an async method so I suppose you want to make this dispatch asynchronous, so you need the await keyword or it won't work correclty